src/main/java/stsjorbsmod/cards/cull/Consumption.java

```diff
@@ -0,0 +1,71 @@
+package stsjorbsmod.cards.cull;
+
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.ui.panels.EnergyPanel;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import stsjorbsmod.JorbsMod;
+import stsjorbsmod.actions.ConsumptionAction;
+import stsjorbsmod.cards.CustomJorbsModCard;
+import stsjorbsmod.characters.Cull;
+
+public class Consumption extends CustomJorbsModCard {
+    public static final String ID = JorbsMod.makeID(Consumption.class);
+    public static final Logger logger = LogManager.getLogger(Cull.class.getName());
+
+
+    private static final CardRarity RARITY = CardRarity.RARE;
+    private static final CardTarget TARGET = CardTarget.ENEMY;
+    private static final CardType TYPE = CardType.ATTACK;
+    public static final CardColor COLOR = Cull.Enums.CULL_CARD_COLOR;
+
+    private static final int COST = -1;
+    private static final int DAMAGE = 1;
+
+    public Consumption() {
+        super(ID, COST, TYPE, COLOR, RARITY, TARGET);
+        baseDamage = DAMAGE;
+        this.isMultiDamage = true;
+        this.exhaust = true;
+    }
+
+    @Override
+    public void use(AbstractPlayer p, AbstractMonster abstractMonster) {
+        int effect = EnergyPanel.totalCount;
```

Comment:
Thanks, I got it working! That's a super useful feature. 
There still 1 thing I'm not sure about: X-cards use this.energyOnUse in the card's use method, but I don't think that will work in calculateBonusBaseDamage. So instead I'm now using EnergyPanel.totalCount. Do you know if there's any behavioural differences between the two that might cause issues down the line?
Let me know if you'd like me to push the update, cheers :)

Reply:
I haven't looked at `energyOnUse` before, but having just now done some code analysis, I think it looks like the primary use case for it is to keep a record of how much energy was allocated towards the use of an X cost card for the sake of play-multiple-copies effects like Burst/Double/etc, so those effects can queue new CardQueueItems for temp cards with energyOnUse set to whatever the original effect used.

I think interacting correctly with effects like that while still showing correct dynamic text may actually be surprisingly complicated, since I think we'd want to use `energyOnUse` when it is set/falling back to `EnergyPanel.totalCount` when it is not set, but the former doesn't normally get reset after use. I'm going to do some experimenting with it.

Reply:
Did some experimenting; I think the best option is going to be to keep the clause you've got here as-is, but move it to calculateBonusBaseDamage, in particular this part:

```java
        int effect = EnergyPanel.totalCount;
        if (this.energyOnUse != -1) {
            effect = this.energyOnUse;
        }
```

...and then add to `CustomJorbsModCard`'s ctor and onMoveToDiscard lines that reset `this.energyOnUse` back to -1. That way, it uses `energyOnUse` if it is available (in particular, in the `DoubleExhuastPower` case), but falls back to using the total count when calculating card descriptions before playing.